apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -27,59 +27,36 @@ private ConnectionPoolOptions()
         }
 
         /// <summary>
-        ///     The default instance of connection pool options.
+        /// The default instance of connection pool options.
         /// </summary>
         public static ConnectionPoolOptions Instance { get; } = new ConnectionPoolOptions();
 
         /// <summary>
         /// </summary>
-        public TimeSpan KeepAliveTimeSpan { get; set; } = TimeSpan.FromMinutes(55);
+        public TimeSpan KeepAliveInterval { get; set; } = TimeSpan.FromMinutes(55);
 
         /// <summary>
-        ///     Sets the log level for diagnostic logs sent to Trace (for desktop) or stderr (for .Net Core).
+        /// Sets the log level for diagnostic logs sent to Trace (for desktop) or stderr (for .Net Core).
         /// </summary>
         public LogLevel LogLevel
         {
             get
             {
-                switch (Logger.LogLevel)
+                int underlyingLevel = (int)Logger.LogLevel;
+                if (Enum.IsDefined(typeof(LogLevel), underlyingLevel))
                 {
-                    case V1.Logging.LogLevel.Debug:
-                        return LogLevel.Debug;
-                    case V1.Logging.LogLevel.Info:
-                        return LogLevel.Info;
-                    case V1.Logging.LogLevel.Warn:
-                        return LogLevel.Warn;
-                    case V1.Logging.LogLevel.Error:
-                        return LogLevel.Error;
-                    case V1.Logging.LogLevel.None:
-                        return LogLevel.None;
-                    default:
-                        throw new ArgumentOutOfRangeException();
+                    return (LogLevel) underlyingLevel;
                 }
+                throw new IndexOutOfRangeException("Invalid log level specified.");
```

Comment:
I think ArgumentOutOfRangeException was more appropriate - this isn't an index into an array or whatever.
(But I do like the compactness...)